hooks/persistence-defectdojo/hook/src/main/java/io/securecodebox/persistence/models/SecureCodeBoxFinding.java

```diff
@@ -42,14 +42,10 @@
   Map<String, Object> attributes;
 
   public enum Severities {
-    @JsonProperty("HIGH")
-    High,
-    @JsonProperty("MEDIUM")
-    Medium,
-    @JsonProperty("LOW")
-    Low,
-    @JsonProperty("INFORMATIONAL")
-    Informational
+    HIGH,
+    MEDIUM,
+    LOW,
+    INFORMATIONAL
```

Comment:
You mentioned that the Jackson annotations were used wrongly.
Was it intentional to remove these annotations all together?

Reply:
I'm not sure what causes this to behave wrongly, as this seems to work when deserializing JSON Findings to java objects but seems to fail during serialization. The docs didn't indicate to me that this would be a problem.

Naming the enums exactly like their actual value still seems like the easiest solution, and is even more consistent with the usual naming schema of java enums.